packages/jest-cli/src/reporters/JestReporterCoverage.js

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
ugh.. i'm going to kill this file in my next PR, because the coverage will basically have to be rewritten anyway :)